Integrations/python/deephaven/learn/__init__.py

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+
+"""
+Deephaven's learn module enables developers and data scientists to use familiar
+"""
+
+import jpy
+import wrapt
+
+_Input_ = None
+_Output_ = None
+_Computer_ = None
+_Scatterer_ = None
+
+
+def _defineSymbols():
+    """
+    Defines appropriate java symbol, which requires that the jvm has been initialized through the :class:`jpy` module,
+    for use throughout the module AT RUNTIME. This is versus static definition upon first import, which would lead to an
+    exception if the jvm wasn't initialized BEFORE importing the module.
```

Comment:
Could be more clear.  Is this copied from elsewhere?

Reply:
Yes, copied from TableTools, I believe. Will try to clarify.